packages/x-date-pickers/src/DateTimePicker/DateTimePickerToolbar.tsx

```diff
@@ -185,6 +185,15 @@ const DateTimePickerToolbarAmPmSelection = styled('div', {
   },
 }));
 
+/**
+ * Demos:
+ *
+ * - [DateTimePicker](https://mui.com/x/react-date-pickers/date-time-picker/)
```

Comment:
Same question regarding the `components` list would also apply here. 🤔 
```suggestion
 * - [DateTimePicker](https://mui.com/x/react-date-pickers/date-time-picker/)
 * - [Custom components](https://mui.com/x/react-date-pickers/custom-components/)
```